config/config.go

```diff
@@ -168,6 +168,8 @@ type SystemConfiguration struct {
 
 		Uid int `yaml:"uid"`
 		Gid int `yaml:"gid"`
+
+		Login bool `yaml:"login"`
```

Comment:
Adding a comment on what this setting does would be useful for people quickly skimming the code.

Should we also allow this to be a string so the user can configure what the login shell should be?  Ideally we would move this to an egg option, but for now this solution should be fine to work around the majority of issues.

Reply:
Honestly this doesn't work the way I thought it would when I tried to manage if the user could ahve a working terminal. If someone sets the start command to a shell it bypasses the users shell.

Reply:
That makes sense as it would only use the user's shell if we trigger a login as the user.  When the container runs it completely ignores the user's shell as we don't override the Command used to start the container, so it uses whatever is defined in the image used for the server (which matters for attaching to the container).  Exec also doesn't respect the user's shell unless you made it "login" as the user, as you are directly executing a command.